### ISSUE TITLE
Canvas Keyboard Navigation

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#10999]
## Purpose

Ensure that basic functionality within editor canvas is working with keyboard ahead of introducing multi-page canvas view in editor. 

### Current issues with keyboard navigation in canvas

- Currently it seems to be impossible to interact with the background using keyboard
- It also seems to be impossible to interact with masked elements (e.g. shapes, all media, etc.) using just keyboard
- As @samwhale  found out during discovery (#10850) - when tabbing from the library, the focus actually goes to a masked element (or background) on the page but since the events are disabled for masked elements, the masked elements do not get selected nor are interactive nor have any indicator that focus is on those elements, making it impossible to do anything with those.
- Floating menu traps focus, so user can’t go back to the canvas.

### Related issues
(These should all be solved residually, these are things to check on after scoped work is completed) 

- [#10793](https://github.com/GoogleForCreators/web-stories-wp/issues/10793)
- [#10698](https://github.com/GoogleForCreators/web-stories-wp/issues/10698)

### High level feature

Two things need to happen here for fixing keyboard navigation of the canvas. 
1. All elements on a canvas should be focusable via keyboard.
2. Floating menu needs to be accessible.

### Existing Documentation 

- https://docs.google.com/document/d/15FHCukTDrWf7xYpPkBG3bGRgRQZeqzkCcglaNXgaw9s/edit#
- https://docs.google.com/document/d/1_3-_v4zH6bW9hOplOO8wYxj8l4C0soMgEbD7XLiXrrg/edit#
- https://docs.google.com/spreadsheets/d/1zE3rdJykQY1YOQUHoczyzPNLa5_bhtv-2wfTE1-NOn0/edit#gid=0

**Comment by BrittanyIRL** [comment]
Running list of todos once focus functionality is fixed:
- https://github.com/GoogleForCreators/web-stories-wp/pull/11215/files#r847459521

**Comment by sblinde** [comment]
Completed, closing this for Brittany.
